guides/getting-started.md

```diff
@@ -61,15 +61,16 @@ import 'hammerjs';
 ```
 
 ## Configuring SystemJS
-If your project is using SystemJS for module loading, you will need to add `@angular/material` 
+If your project is using SystemJS for module loading, you will need to add `@angular/material` and `hammerjs`
 to the SystemJS configuration:
 
 ```js
 System.config({
   // existing configuration options
   map: {
     ...,
-    '@angular/material': 'npm:@angular/material/bundles/material.umd.js'
+    '@angular/material': 'npm:@angular/material/bundles/material.umd.js',
+    'hammerjs': 'npm:hammerjs/hammer.js'
```

Comment:
Since there's more than one way to include hammer, I'd rather leave this example the same and then add another block under it that says something like 

"If you're loading HammerJS via npm as well, it also needs to be added to the SystemJS config: "